go.mod

```diff
@@ -113,33 +113,4 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace (
-	k8s.io/api => k8s.io/api v0.26.0
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.26.0
-	k8s.io/apimachinery => k8s.io/apimachinery v0.26.0
-	k8s.io/apiserver => k8s.io/apiserver v0.26.0
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.26.0
-	k8s.io/client-go => k8s.io/client-go v0.26.0
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.26.0
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.26.0
-	k8s.io/code-generator => k8s.io/code-generator v0.26.0
-	k8s.io/component-base => k8s.io/component-base v0.26.0
-	k8s.io/component-helpers => k8s.io/component-helpers v0.26.0
-	k8s.io/controller-manager => k8s.io/controller-manager v0.26.0
-	k8s.io/cri-api => k8s.io/cri-api v0.26.0
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.26.0
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.26.0
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.26.0
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.26.0
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.26.0
-	k8s.io/kubectl => k8s.io/kubectl v0.26.0
-	k8s.io/kubelet => k8s.io/kubelet v0.26.0
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.26.0
-	k8s.io/metrics => k8s.io/metrics v0.26.0
-	k8s.io/mount-utils => k8s.io/mount-utils v0.26.0
-	k8s.io/node-api => k8s.io/node-api v0.26.0
-	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.26.0
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.26.0
-	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.26.0
-	k8s.io/sample-controller => k8s.io/sample-controller v0.26.0
-)
+replace k8s.io/client-go => k8s.io/client-go v0.26.0
```

Comment:
I think even this can be removed

Reply:
Can you try setting 0.26.0 in L32?

Reply:
Oh I think that worked. I'll try to remove that for PDCSI also.

Reply:
Changing to 0.26.0 and removing the replace statement results in 

```
go: sigs.k8s.io/gcp-filestore-csi-driver/pkg/client/clientset/versioned imports
        k8s.io/client-go/util/flowcontrol imports
        k8s.io/apimachinery/pkg/util/clock: module k8s.io/apimachinery@latest found (v0.31.0), but does not contain package k8s.io/apimachinery/pkg/util/clock
go: sigs.k8s.io/gcp-filestore-csi-driver/pkg/csi_driver imports
        k8s.io/client-go/informers imports
        k8s.io/api/auditregistration/v1alpha1: module k8s.io/api@latest found (v0.31.0), but does not contain package k8s.io/api/auditregistration/v1alpha1
go: sigs.k8s.io/gcp-filestore-csi-driver/pkg/csi_driver imports
        k8s.io/client-go/informers imports
        k8s.io/api/batch/v2alpha1: module k8s.io/api@latest found (v0.31.0), but does not contain package k8s.io/api/batch/v2alpha1
go: sigs.k8s.io/gcp-filestore-csi-driver/pkg/csi_driver imports
        k8s.io/client-go/informers imports
        k8s.io/api/settings/v1alpha1: module k8s.io/api@latest found (v0.31.0), but does not contain package k8s.io/api/settings/v1alpha1
go: sigs.k8s.io/gcp-filestore-csi-driver/pkg/webhook imports
        sigs.k8s.io/controller-runtime/pkg/certwatcher tested by
        sigs.k8s.io/controller-runtime/pkg/certwatcher.test imports
        sigs.k8s.io/controller-runtime imports
        sigs.k8s.io/controller-runtime/pkg/builder imports
        sigs.k8s.io/controller-runtime/pkg/client imports
        k8s.io/client-go/metadata: package k8s.io/client-go/metadata provided by k8s.io/client-go at latest version v0.31.0 but not at required version v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
```

I also tried v0.31.0, but it keeps saying it wants k8s.io/client-go/metadata at v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible

Reply:
bummer, this is still a good improvement though so let's go with what we have for now